docs/src/comparisons.md

```
@@ -1,52 +1,58 @@
-  - Elasti takes itself out of the http path once the service has been scaled up.
+    - KEDA http add-on inserts itself in the http path and handles requests even when the service has been scaled up.
+    - KubeElasti takes itself out of the http path once the service has been scaled up.
 - **Integration**:
-  - KEDA HTTP Add-on requires KEDA installation and configuration.
-  - Elasti can work standalone or integrate with KEDA if needed.
+    - KEDA HTTP Add-on requires KEDA installation and configuration.
+    - KubeElasti can work standalone or integrate with KEDA if needed.
 
-## Feature Comparison Table
+## **Feature Comparison Table**
 
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: div

(MD033, no-inline-html)